NEWS.md

```diff
@@ -1,6 +1,10 @@
 
 # vctrs 0.2.0.9000
 
+* `vec_ptype()` has relaxed default behaviour for base types; now if two
```

Comment:
This feels more like a bug fix than a new rule? Since we normally have inheritance of common type methods.

Or perhaps it's just a workaround for the limitations our subtyping rule with container types (I'm starting to think of arrays/matrices as a separate type hierarchy).

 (No need to change the NEWS bullet, just to make sure I understand correctly)

Reply:
Right. It's more of a bug fix, and I agree think that arrays are a separate hierarchy.